src/app/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -1,151 +1,218 @@
+              </p>
+              <p>
+                Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+              </p>
+              <p>
+                Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+              </p>
+              <p>
+                Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+              </p>
+              <p>
+                Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+              </p>
+              <p>
+                Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+              </p>
+              <p>
+                Work Items <a>#201</a> assigned to you by <a>@magranfie</a> under wireframe label
+              </p>
+              </div>
             </div>
+          </div>
+          <div class="card-pf-footer">
+            <a class="margin-bottom-10">
+              <i class="pficon pficon-add-circle-o"></i>
+              <span>Add a Codebase</span>
+            </a>
+          </div>
         </div>
-        <div class="col-md-2">
-            <div class="panel panel-300 panel-default clickable" (click)="router.navigate(['/', space.relationalData.creator.attributes.username, space.attributes.name]);">
-
-                <div class="panel-heading">
-                    <h2 class="panel-title text-center">Hypotheses Evaluating</h2>
-                </div>
-                <div class="panel-body">
-
-                    <p class="text-center number">0</p>
-                </div>
-
+      </div>
+    </div>
+    <div class="col-md-4">
+      <div class="card-pf">
+        <div class="flex-container in-column-direction">
+          <h2 class="card-pf-title card-pf-utilization-details">
+            <span>Pipelines</span>
+            <span class="badge">0</span>
+          </h2>
+          <div class="card-pf-body">
+            <div class="blank-slate-pf">
+              <div class="blank-slate-pf-icon">
+                <span class="fa fa-code-fork"></span>
+              </div>
+              <h1>There are no pipelines executing</h1>
+              <p>
+                This will update once you commit and build your codebase.
+                Click the 'Refresh' button to see the latest updates.
+              </p>
+              <div class="blank-slate-pf-main-action">
+                <button class="btn btn-primary btn-lg">Refresh</button>
+              </div>
+            </div>
+            <div class="analyze-overview-items-detail">
+              <div class="card-pf-body">
+                <p>
+                  <i>Pipeline execution updates are provided here.</i>
+                </p>
+              </div>
             </div>
+          </div>
+          <div class="card-pf-footer">
+            <a class="margin-bottom-10">
+              &nbsp;&nbsp;&nbsp;
```

Comment:
Why did you use this instead of some place holder text or some css if you don't want text?